src/main/java/com/twingly/search/Query.java

```diff
@@ -0,0 +1,157 @@
+package com.twingly.search;
+
+import com.twingly.search.exception.TwinglyException;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * The type Query.
+ */
+public class Query {
+    private static final String BASE_URL = "http://api.twingly.com";
```

Comment:
I see, Betamax will use what the Twingly Search API client uses. 

Exactly what are the problem with HTTPS and Betamax? Is it something with our SSL setup on the server-side?

Just looking at "IMPLEMENTATIONS" at http://betamax.software/ sounds like it should work with both implementations they present.